actions/gha-workflow-validator/src/utils.ts

```diff
@@ -1,27 +1,72 @@
 import * as core from "@actions/core";
+import * as glob from "@actions/glob";
+import { sep } from "path";
+import { readFile } from "fs/promises";
```

Comment:
is there a reason to use `readFile` instead of `readFileSync`?

i think the latter is faster?

https://gist.github.com/FauxFaux/079dc1c696cd60c8120a5ed9b18942c3

Reply:
Will update in #593